opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/aggregation/AggregationQueryBuilder.java

```diff
@@ -112,10 +112,12 @@ public Map<String, OpenSearchDataType> buildTypeMapping(
       List<NamedAggregator> namedAggregatorList,
       List<NamedExpression> groupByList) {
     ImmutableMap.Builder<String, OpenSearchDataType> builder = new ImmutableMap.Builder<>();
-    namedAggregatorList.forEach(agg -> builder.put(agg.getName(),
-        OpenSearchDataType.of(agg.type())));
-    groupByList.forEach(group -> builder.put(group.getNameOrAlias(),
-        OpenSearchDataType.of(group.type())));
+    namedAggregatorList.forEach(agg -> {
```

Comment:
Changes became no-op with last fix, you can revert them completely I think.

Reply:
this is just a whitespace change now... this is fine to keep as is.

Reply:
Reverted in https://github.com/opensearch-project/sql/pull/1664/commits/1591a85db1e5627db9601b44b5e3d66079a9138f